pandas/core/series.py

```diff
@@ -887,21 +887,18 @@ def __getitem__(self, key):
         elif key_is_scalar:
             return self._get_value(key)
 
-        if (
-            isinstance(key, tuple)
-            and is_hashable(key)
-            and isinstance(self.index, MultiIndex)
-        ):
+        if is_hashable(key):
             # Otherwise index.get_value will raise InvalidIndexError
```

Comment:
can you add a comment here about what types we expect to be relevant?  i.e. scalars are already handled above, so we just expect tuple and frozenset or similar